modules/authorization/role-assignments/metadata.json

```diff
@@ -0,0 +1,6 @@
+{
+  "$schema": "https://aka.ms/bicep-registry-module-metadata-file-schema-v2#",
+  "name": "Role Assignments",
+  "summary": "This module deploys Role Assignments across the management group, subscription or resource group scope.",
```

Comment:
Should the README.md be updated to "This module deploys Role Assignments at a Management Group, Subscription, or Resource Group scope." and then this file updated accordingly?